container/test-impl-base/src/main/java/org/jboss/arquillian/container/test/impl/client/container/ClientContainerController.java

```diff
@@ -48,16 +48,16 @@
 public class ClientContainerController implements ContainerController
 {
 
-   private final Logger log = Logger.getLogger(ClientContainerController.class.getName());
+   protected final Logger log = Logger.getLogger(ClientContainerController.class.getName());
```

Comment:
Haha, yeah, this is wrong, but i do have one in the POC just kept this wrongly...